org-roam.el

```diff
@@ -706,6 +700,15 @@ If there is no file with that ref, a file with that ref is created.
 
 This function is used solely in Org-roam's capture templates: see
 `org-roam-capture-templates'."
+  (org-capture-put :template
+                   (concat
+                    (org-roam--fill-template (or (org-capture-get :head)
+                                                 org-roam--capture-header-default)
+                                             org-roam--capture-info)
+                    "\n" (org-capture-get :template)))
```

Comment:
I like this change, this means that if org-capture is cancelled, the head isn't written.